### PR TITLE
Edit Rust SDK doc, add links to supporting docs

### PR DESCRIPTION
**Pull request by Clay-Mysten**

### CODE REVIEW

File: doc/src/build/rust-sdk.md
Comment (randall-Mysten) [inline doc/src/build/rust-sdk.md:32]: this is intentionally public viewable?

**Review by randall-Mysten — APPROVED** [review]
Approving with a comment/question about the To Do line

File: doc/src/build/rust-sdk.md
Comment (Clay-Mysten) [reply doc/src/build/rust-sdk.md:32]: > this is intentionally public viewable?

Yep. Let's have Patrick review before merge.

Comment (patrickkuo) [reply doc/src/build/rust-sdk.md:32]: I am expecting the audience of this doc knows the basic of Rust and `Cargo.toml`, I don't think we need more descriptions here?

Comment (Clay-Mysten) [reply doc/src/build/rust-sdk.md]: > I am expecting the audience of this doc knows the basic of Rust and `Cargo.toml`, I don't think we need more descriptions here?

Great. I removed the TODO and linked into the Cargo doc.

**Pull request merged**
